content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -219,7 +219,7 @@ yum install -y kubeadm-1.19.x-0 --disableexcludes=kubernetes
 
   You can now apply the upgrade by executing the following command:
 
-      kubeadm upgrade apply v1.19.0
+      kubeadm upgrade apply v1.19.0 --etcd-upgrade=false
```

Comment:
Thanks for your contribution @632661265 , but it will be appreciated if you can synchronize all the changes from https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

Note: In the latest version of the document, this paragraph has been removed.